docs/operators_and_staking/staking.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Staking guide
+sidebar_position: 2
+description: Staking guide
+keywords:
+- Staking guide
+- Staking
+---
+
+### Select an operator to nominate 
+
+:::note
+Three important factors to pay attention to are `minimumNominatorStake`, `nominationTax`, and `status`.
+:::
+
+1. Visit PolkadotJS **[Network Explorer](https://polkadot.js.org/apps/#/explorer)**.
+2. Go to `Developer` -> `Chain state`.
+
+ ![Staking-16](/img/doc-imgs/operators-staking/Staking-16.png)
+3. Select `domains` under `selected state query` and choose `operators`, exclude `option` and click on `+` to run the query.
+4. Browse the list of available operators, make sure the status is `Registered` and that `minimumNominatorStake` is lower than your staking amount. 
+
+ ![Staking-17](/img/doc-imgs/operators-staking/Staking-17.png)
+
+:::note
+Number 17 on the screenshot above corresponds to `operatorID`.
```

Comment:
operatorID -> operatorId

Worth doing a case sensitive serach for this as it's in a few places.